File: Python Code/Daily_Flash_Code_0/Week 5/11 Feb/25-DailyFlash_Solutions/25-DailyFlash_Solutions/11_Feb_Solution_Three/C/prog3.c
#include<stdio.h>

int main(void) {
	
	int num;
	printf("Enter num : ");
	scanf("%d", &num);
	while(num >= 0) {
		if(num % 2 == 0)
			printf("%d ", num);
		num--;
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 5/11 Feb/25-DailyFlash_Solutions/25-DailyFlash_Solutions/11_Feb_Solution_Three/Java/prog3.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		
		int num;
		System.out.println("Enter num : ");
		Scanner sc = new Scanner(System.in);
		num = sc.nextInt();
		while(num >= 0) {
			if(num % 2 == 0)
				System.out.print(num + " ");
			num--;
		}
	}
}
